engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/TestMessages.scala

```diff
@@ -109,4 +110,84 @@ object TestMessages {
         )
       )
     )
+
+  /** Create an error update response.
+    *
+    * @param contextId an identifier of the context
+    * @param expressionId an identifier of the expression
+    * @param payload the error payload
+    * @return the expression update response
+    */
+  def error(
+    contextId: UUID,
+    expressionId: UUID,
+    payload: Api.ExpressionUpdate.Payload
+  ): Api.Response =
+    error(contextId, expressionId, Constants.ERROR, payload)
+
+  /** Create an error update response.
+    *
+    * @param contextId an identifier of the context
+    * @param expressionId an identifier of the expression
+    * @param expressionType a type of the expression
+    * @param payload the error payload
+    * @return the expression update response
+    */
+  def error(
+    contextId: UUID,
+    expressionId: UUID,
+    expressionType: String,
+    payload: Api.ExpressionUpdate.Payload
+  ): Api.Response =
+    Api.Response(
+      Api.ExpressionUpdates(
+        contextId,
+        Set(
+          Api.ExpressionUpdate(
+            expressionId,
+            Some(expressionType),
+            None,
+            Vector(Api.ProfilingInfo.ExecutionTime(0)),
+            false,
+            payload
+          )
+        )
+      )
+    )
```

Comment:
This overload should delegate to the one below.

Reply:
Unfortunately, we can't because the one below requires a `methodPointer`, and here we pass `None`.